tests/wamr-test-suites/wasi-test-script/run_wasi_tests.sh

```diff
@@ -96,7 +112,13 @@ else
     for testsuite in ${THREAD_STRESS_TESTS} ${THREAD_PROPOSAL_TESTS} ${THREAD_INTERNAL_TESTS}; do
         tests=$(ls ${testsuite}*.wasm)
         tests_array=($tests)
-        run_aot_tests "${tests_array[@]}"
+
+        readarray -t excluded_tests_array < <(jq -c \
+            --slurpfile testsuite_manifest $testsuite/manifest.json \
+            '.[$testsuite_manifest[0].name] // {} | keys[]' \
+            $TEST_FILTER)
```

Comment:
It's been raised [here](https://github.com/bytecodealliance/wasm-micro-runtime/issues/2536) already but as a future improvement we could alter the wasi test runner script to add AOT support rather than duplicating the test runner logic (like filtering etc.) here.